liquibase-couchbase/src/test/java/common/matchers/CouchbaseCollectionAssert.java

```diff
@@ -82,14 +85,34 @@ public CouchbaseCollectionAssert contains(@NonNull List<Document> docs) {
     }
 
     public CouchbaseCollectionAssert contains(Document doc) {
-        extractingDocument(doc.getId()).itsContentEquals(doc.getContentAsJson());
+        extractingDocument(doc.getId(), getClassName(doc.getValue())).itsContentEquals(doc.getValue());
 
         return this;
     }
+
     public CouchbaseDocumentAssert containsDocument(Document doc) {
-        extractingDocument(doc.getId()).itsContentEquals(doc.getContentAsJson());
+        extractingDocument(doc.getId(), getClassName(doc.getValue())).itsContentEquals(doc.getContentAsJson());
+
+        return extractingDocument(doc.getId(), getClassName(doc.getValue()));
+    }
 
-        return extractingDocument(doc.getId());
+    private Class getClassName(Value value) {
```

Comment:
Looks like this method could be moved to `Value` class itself

Reply:
I don't agree. Because this logic with Class object is only needed in tests and assertions. We don't need it in other cases

Reply:
Minor: point to discuss, what do other reviewers think about it? (if it will be the last point - could be closed without any doubts)